Package@swift-5.swift

```diff
@@ -0,0 +1,19 @@
+// swift-tools-version:5.0
+
+import PackageDescription
+
+let package = Package(
+    name: "Commandant",
+    products: [
+        .library(name: "Commandant", targets: ["Commandant"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/Quick/Quick.git", from: "2.0.0"),
+        .package(url: "https://github.com/Quick/Nimble.git", from: "8.0.0")
+    ],
+    targets: [
+        .target(name: "Commandant", dependencies: []),
+        .testTarget(name: "CommandantTests", dependencies: ["Commandant", "Quick", "Nimble"])
+    ],
+    swiftLanguageVersions: [.version("5")]
```

Comment:
you can use `.v5` https://github.com/apple/swift-package-manager/blob/dc27f4bc98324e54b7c95169c7afe1ed9455f59b/Sources/PackageDescription4/ManifestVersion.swift#L17

```suggestion
    swiftLanguageVersions: [.v5]
```

Reply:
https://github.com/apple/swift-package-manager/blob/dc27f4bc98324e54b7c95169c7afe1ed9455f59b/Sources/PackageModel/ToolsVersion.swift#L139